.github/workflows/cmake.yml

```diff
@@ -87,29 +87,34 @@ jobs:
         ${{ toJSON(github.event) }}
         EOF
 
+    - name: Generate IRC message
+      if: always()
+      id: ircmsg
+      run: |
+        ${{ github.workspace }}/Meta/mangle_event_for_irc.py <<"EOF"
+        ["${{ github.actor }}", ${{ github.run_id }}, "${{ job.status }}",
+        ${{ toJSON(github.event) }}
+        ]
+        EOF
+
     - name: IRC result notification (direct push)
       uses: rectalogic/notify-irc@v1
       if: github.repository == 'SerenityOS/serenity' && github.ref == 'refs/heads/master' && github.event_name == 'push' && !cancelled()
       with:
-        channel: "#serenityos"
+        channel: "#irc-ga-test"
```

Comment:
Committing this was probably not intentional ;)

Reply:
Le gasp, my secret test channel has been leaked! Yup, I need to change that.